bin/satellite-clone.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+from optparse import OptionParser
+import os
+import subprocess
+
+def yes_or_no(question):
+    while "the answer is invalid":
```

Comment:
You mean `while True`? :)

Reply:
technically a non-empty string is trueish so it will work :)

Reply:
having the string does explicitly explain the condition of the loop, maybe keep it for this reason?